guizero/Painter.py

```diff
@@ -0,0 +1,26 @@
+from tkinter import Canvas, ALL
+from . import utilities as utils
+
+
+class Painter:
+    def __init__(self,master,width=100,height=100,background='white',grid=None,align=None):
+
+        self.tk = Canvas(master.tk,width=width,height=height,background=background)
+        self._height = height
+        self._width = width
+        
```

Comment:
the class should preserve its master, grid and align attributes, so they are accessible to guizero through the mixins